src/lib/sedlexing.ml

```diff
@@ -471,7 +450,15 @@ module Utf8 = struct
   let from_gen s =
     create (fill_buf_from_gen (fun id -> id) (Helper.gen_from_char_gen s))
 
-  let from_string s = from_int_array (Helper.to_int_array s 0 (String.length s))
+  let from_string s =
+    let gen =
```

Comment:
This can be implemented with `Gen.init ~limit:(String.length s) (fun i -> String.get s i)`

Reply:
Done thanks.